changelog/differ_test.go

```diff
@@ -3,6 +3,8 @@ package main
 import (
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
```

Comment:
Please run `fmt-imports`, this change (and others) will appear with next code generation.

Reply:
This is the result of formatting after the `fmt-imports` command.